WORKSPACE

```diff
@@ -20,7 +20,8 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #android_sdk_repository(
 #    name = "androidsdk",
 #    api_level = 23,
-#    build_tools_version = "25.0.1",
+#    build_tools_version = "25.0.2",
+#    #Check your build tools version. Android Studio is shipping with 25.0.2 currently.
```

Comment:
Is this correct? I see 25.0.4 in Android Studio now.

In any case I would:
1 ) move the comment above the build_tools_version line
2) rephrase it to say "# Ensure that you have a matching version of build_tools installed, as the SDK default changes periodically."